filament/src/materials/dof.mat

```diff
@@ -445,32 +469,31 @@ void postProcess(inout PostProcessInputs postProcess) {
             foreground *= rcp(c);
             fgOpacity  *= rcp(sampleCount(ringCountGather));
         }
+        //foreground.r += 0.1;
```

Comment:
Is this supposed to be commented? Should it be removed?

Reply:
There are 3 of those, I use them for debugging with matdbg. It's basically you're first line of defense when something goes wrong, as it tells you in which type of tile the problem lies.

Reply:
You should leave an explanation so we don't get rid of them or stare at them wondering later on :)